File: H_Spell_Check.cpp
#include<bits/stdc++.h>
using namespace std;

int main(void){
    int n;
    cin>>n;
    while(n--){
        int x;
        cin>>x;
        string s;
        cin>>s;
        sort(s.begin(), s.end());
        string nam = "Timur";
        sort(nam.begin(),nam.end());
        if(nam == s){
            cout<<"YES"<<endl;
        }
        else{
            cout<<"NO"<<endl;
        }
    }
    return 0;
}

File: H_Spell_Check.py

for _ in range(int(input())):
    n = int(input())
    user_name = input()
    real_name = "Timur"

    if( n != 5 ):
        print("NO")
    else:
        if( set(user_name) == set(real_name) ):
            print("YES")
        else:
            print("NO")